app/src/main/java/net/gsantner/opoc/frontend/base/GsFragmentBase.java

```diff
@@ -45,6 +45,9 @@
     protected Menu _fragmentMenu = new GsMenuItemDummy.Menu();
     protected Queue<Runnable> _postTasks = new LinkedList<>();
 
+    private final static String EXTRA_IDENTIFIER_KEY = "GS_FRAGMENT_BASE_EXTRA_IDENTIFIER";
+    private String _extraIdentifier = null;
```

Comment:
System for setting an extra identifier / tag in a GsFrament

Reply:
Removed as not needed. Turns out FragmentManager has a built in mechanism for just this